task1/test_task1.py

```diff
@@ -0,0 +1,62 @@
+import random
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+import time
+import pytest
+from pprint import pprint
+
```

Comment:
Double check if all the imports are necessary

Reply:
Removed unnecessary imports